.github/drafts/16521/Code-scanner-Draft.md

```diff
@@ -0,0 +1,121 @@
+
+Barcodes are now found on a wide range of goods, including health bands, rental cars, checked baggage, and clothing purchased from supermarkets or retail stores. They each identify a thing or a person and encode significant information.
```

Comment:
Can we start the sentence with `Barcodes and QR codes` instead of just `Barcodes`? From a user's POV who is reading this doc for the first time and only taking a quick glance, they might think that this widget is only for barcodes.